### ISSUE TITLE
Missing "Back to search results" link

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/clinical-trials-search-app#633]
### Issue description
When navigating from a search results list to a trial detail page, the details page should display a "Back to search results" link.  This link is currently missing.

Interestingly, configuring `www-dev-ac` to use the same version of the application as production, the link appears.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1. Go to https://www.cancer.gov/research/participate/clinical-trials-search/r?loc=0&rl=2
2. Open the browser's developer tools and switch to the Network tab.
3. Filter for JS files only.
4. Click on any of the trial links.

### What's the expected result?
- The dev tools should not show any traffic.
- The trial details page should show a "Back to search results" link.

### What's the actual result?
- The dev tools show that `main.js` is reloaded (and other JavaScript too).
- The "Back to search results" link is missing.

**Comment by bryanpizzillo** [comment]
In trying to make R4R generic, I noticed this issue. This is because of a Launch click handler for tracking “legacy components” on mini landing pages.

Basically we thought that if we had a condition for the page type (cgvMiniLanding) that the click event handler would not be attached to items outside of that type. Unfortunately, the click handler is added to all links matching the css selector, and handling the event it makes sure it is a mini landing page.

The unfortunate part is in order to ensure the analytics were handling clicks to other pages we had to use a launch setting for the event that delays the event handling. Adobe basically prevents default, adds a timeout, and when the timeout finishes it sets window.location. Because this happens like this, that link becomes a whole new page load, which reloads the react app, which means the state goes away. For R4R the app’s state is where the search information is stored. For CTS it should be the query params, but there is probably something in the state that is not noticing.

So for R4R I am 100% sure this is the issue. For CTS we would have to check after the Adobe rule is disabled.
